eZ/Publish/SPI/Limitation/Target/Builder/VersionBuilder.php

```diff
@@ -126,4 +126,26 @@ function (Field $field) {
 
         return $this;
     }
+
+    /**
+     * Set intent to publish, to specified languages, all from the given list.
+     *
+     * @param array $languageCodes
+     *
+     * @return self
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     */
+    public function publishLanguages(array $languageCodes): self
```

Comment:
We're not publishing a language but a translation.
```suggestion
    public function publishTranslations(array $languageCodes): self
```